backup-commons/src/main/java/com/opsgenie/tools/backup/retrieval/ScheduleRetriever.java

```diff
@@ -56,6 +58,43 @@ public ScheduleRetriever(boolean includeOverrides) {
 
     }
 
+    private void sortScheduleTimeRestrictions(List<Schedule> schedules) {
+        for(Schedule schedule: schedules) {
+            for(ScheduleRotation scheduleRotation: schedule.getRotations()) {
+                if(scheduleRotation.getTimeRestriction() instanceof WeekdayTimeRestrictionInterval) {
```

Comment:
`scheduleRotation.getTimeRestriction()` can return `null` in case there are no restrictions for the given rotation. `null instanceof WeekdayTimeRestrictionInterval` would evaluate to `false` in that case.

I can add an explicit check for `null` as well to indicate the presence of that case for readability if that makes more sense.